react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,32 @@
+/* eslint-disable no-console */
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (!isArray(args)) {
+    return args;
+  } else if (args.length === 0) {
+    return undefined;
+  } else if (args.length === 1) {
```

Comment:
I don't really get why you do this check. I believe this does not take into consideration nested arrays, and if that were the case I think we should check for every element in the array. 
Don't the tests pass if you remove this case?

Reply:
no, this test fails: 
`it('min can return the minimum based on a list', () => {
    expect(min([1, 2])).toBe(1);
    expect(min([2, 1])).toBe(1);
  });`

because the min method receives an array like this `[[1, 2]]`

Reply:
Okay I see what you did, I think line 11's spread is unnecessary but it doesnt matter. Nice job!